common/src/main/java/io/druid/metadata/MetadataStorageTablesConfig.java

```diff
@@ -67,6 +73,8 @@ public static MetadataStorageTablesConfig fromBase(String base)
   @JsonCreator
   public MetadataStorageTablesConfig(
       @JsonProperty("base") String base,
+      @JsonProperty("dataSource") String dataSourceTable,
```

Comment:
This list is getting kind of silly... we may want to consider a better way of handling this.